docs/examples/image-classification.md

```
@@ -0,0 +1,685 @@
+```
+
+### Step 3: Run Training
+
+```bash
+cd cifar10
+lighter fit configs/resnet18.yaml
+```
+
+**Expected output:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)